src/_P062_MPR121_KeyPad.ino

```diff
@@ -74,6 +81,100 @@ boolean Plugin_062(byte function, struct EventStruct *event, String& string)
     {
       addFormCheckBox(F("ScanCode"), F("scancode"), PCONFIG(1));
 
+      addFormSubHeader(F("Sensitivity"));
+      {
+        uint8_t touch_treshold   = PCONFIG(2);
+        if(touch_treshold == 0) {
+          touch_treshold = P062_DEFAULT_TOUCH_TRESHOLD; //default value
+        }
+        addFormNumericBox(F("Touch treshold (1..255)"), F("p062_touch_treshold"), touch_treshold, 0, 255);
+        String unit_ = F("Default: ");
+        unit_ += String(P062_DEFAULT_TOUCH_TRESHOLD);
+        addUnit(unit_);
+      }
+
+      {
+        uint8_t release_treshold = PCONFIG(3);
+        if(release_treshold == 0) {
+          release_treshold = P062_DEFAULT_RELEASE_TRESHOLD; //default value
+        }
+        addFormNumericBox(F("Release treshold (1..255)"), F("p062_release_treshold"), release_treshold, 0, 255);
+        String unit_ = F("Default: ");
+        unit_ += String(P062_DEFAULT_RELEASE_TRESHOLD);
+        addUnit(unit_);
+      }
+      {
+        bool canCalibrate = true;
+        bool tbUseCalibration = bitRead(P062_CONFIG_FLAGS, P062_FLAGS_USE_CALIBRATION);
+
+        P062_data_struct *P062_data = static_cast<P062_data_struct *>(getPluginTaskData(event->TaskIndex));
+
+#ifdef PLUGIN_062_DEBUG
+        String log = F("p062_data size: ");
+        log += sizeof(P062_data);

```

Comment:
This will return "4". (the size of a pointer)
You probably meant `sizeof(P062_data_struct)` or `sizeof(*P062_data)`

Reply:
Yeah, that's code I intended to remove but forgot to do. Will do soon.